com.blitterstudio.amiberry.yml

```diff
@@ -0,0 +1,101 @@
+app-id: com.blitterstudio.amiberry
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: amiberry.sh
+rename-icon: amiberry
+copy-icon: true
+finish-args:
+  - --device=all
+  # X11 + XShm access
+  - --share=ipc
+  # Wayland access
+  - --socket=wayland
+  - --socket=fallback-x11
+  # Needs to talk to the network:
+  - --share=network
+  # Allow bluetooth (e.g. controllers)
+  - --allow=bluetooth
+  # Play sounds using PulseAudio
+  - --socket=pulseaudio
+  # Filesystem access
+  - --filesystem=host
+  
+modules:
+  - name: libmpeg2
+    config-opts:
+      - --disable-static
+    rm-configure: true
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - "*.la"
+      - "*.a"
+    sources:
+      - type: archive
+        url: http://libmpeg2.sourceforge.net/files/libmpeg2-0.5.1.tar.gz
+        sha256: dee22e893cb5fc2b2b6ebd60b88478ab8556cb3b93f9a0d7ce8f3b61851871d4
+      - type: script
+        dest-filename: autogen.sh
+        commands:
+          - autoreconf -vfi
+
+  - name: libserialport
+    buildsystem: autotools
+    cleanup:
+      - /include
+      - "*.la"
+      - "*.a"
+    sources:
+      - type: archive
+        url: https://github.com/BlitterStudio/amiberry/raw/master/external/libserialport/libserialport-0.1.1.tar.gz
+        sha256: 4e3573e3bdbb6b0a872f32c4826d1b6b630dcc3eeb5f05d0501bb79d738a09ea
+
+  - name: amiberry
+    only-arches:
+      - x86_64
+    post-install:
+      - install -Dp -m 755 amiberry /app/bin/amiberry
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    builddir: true
+    
+    sources:
+      - type: archive
+        url: https://github.com/BlitterStudio/amiberry/archive/refs/tags/v5.6.2.tar.gz
+        sha256: 4e9feb3ab0039871a25dda9f42a29c9ec1d6160ff68ef7c37fd3205f59c14b97
+
+  - name: desktop
```

Comment:
this is no reason to have a separate module for this.

Reply:
Thanks, fixed